### PR TITLE
Cascading Colors in Sass Module Syntax

### PR DESCRIPTION
**Pull request by mirisuzanne**
kitten rainbow unicorn

## Description
- Use Sass module syntax
- Setup Cascading Color Systems
- Setup Accoutrement & Herman

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@jgerigmeyer @SondraE @stacyk I think this is ready for review, but it likely requires some conversations since I've been out here making it all up as I go.

- Sondra/Stacy: we should talk about how we want to handle color systems in design
- Stacy: we should talk about how we combine CSS variables with Accoutrement Tokens
- Jonny: I wonder if there's a quick way to tell Herman we don't need examples compiled, in order to silence all the warnings?

Happy to walk through it with you all after a standup. I'm going to move on to building the layout based on this branch…

**Comment by mirisuzanne** [comment]
[How should we use CCS (slider) colors?](https://trello.com/c/WWOtKuHg/8-how-should-we-use-ccs-slider-colors)

File: src/scss/initial/_themes.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// OddBird
+// -------
+/// The OddBird theme is based on default settings,
+/// but removes user hue settings.
+/// @name Default Theme
+/// @group init-themes
+[data-ccs-theme='oddbird'] {
+  --ccs-prime--theme: var(--ccs-prime--config);
+  --ccs-accent--theme: var(--ccs-accent--config);
+  --ccs-custom-hue: none;
```
Comment (stacyk) [inline src/scss/initial/_themes.scss]: It removes the hue setting, but if you switch to another theme, change the hue, and switch back it still keeps the active hue of whatever theme I as just looking at, right? Until you hit reset anyway.

File: src/scss/initial/_type.scss [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
-:root {
-  @include font-family('body');
-  font-size: size('root');
-  font-size: calc(1em + 0.3vw);
-  line-height: 1.35;
+.intro {
+  font-size: var(--medium);
 }
 
 // Selection
```
Comment (stacyk) [inline src/scss/initial/_type.scss]: Are these comments still applicable?

File: src/scss/config/_scale.scss [hunk 3]
```diff
@@ -212,10 +108,10 @@ $column-sizes: (
   'wide-column': 90%,
 );
 
-@include add-sizes($column-sizes);
-@include herman-add(
+@include tools.add-sizes($column-sizes);
```
Comment (stacyk) [inline src/scss/config/_scale.scss]: I'm so not used to this new Sass way of writing this and all the @use statements in all these files. I feel like it reminds me of these Vue files though in that way. or nunjucks.

File: content/_includes/site/ccs.njk [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <details data-ccs="menu" hidden>
   <summary class="menu-item">colors</summary>
   <div data-dropdown="pad">
-    <div class="color-btns">
+    <div data-ccs="buttons">
       <button
         data-ccs-input="mode"
         data-btn="small"
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/site/ccs.njk]: yeah, improved this some on the `nav` branch, and still need to get form elements (like buttons) styled…

File: src/scss/initial/_themes.scss (hunk 1 above)
Comment (mirisuzanne) [reply src/scss/initial/_themes.scss]: that's a bug! fixing…

File: src/scss/initial/_type.scss (hunk 2 above)
Comment (mirisuzanne) [reply src/scss/initial/_type.scss]: oops, updating…

File: src/scss/config/_scale.scss (hunk 3 above)
Comment (mirisuzanne) [reply src/scss/config/_scale.scss]: Yeah, it's a actually based on the Python module system, which is what jinja/nunjucks use - so very similar. :)

**Pull request merged**